examples/scripts/uptime.py

```diff
@@ -53,7 +53,7 @@ def v2_runner_on_failed(self, result, *args, **kwargs):
 def main():
     host_list = ['localhost', 'www.example.com', 'www.google.com']
     # since the API is constructed for CLI it expects certain options to always be set in the context object
-    context.CLIARGS = ImmutableDict(connection='smart', module_path=['/to/mymodules', '/usr/share/ansible'], forks=10, become=None,
+    context.CLIARGS = ImmutableDict(connection='smart', module_path=['/to/mymodules', '/usr/share/ansible'], forks=10, become=False,
```

Comment:
FWIW this doesn't seem to be a problem anymore, nor is it a proper solution.

Reply:
FYI https://github.com/ansible/ansible/pull/74438